src/Arrays/MPIStateArrays.jl

```diff
@@ -523,7 +523,7 @@ function LinearAlgebra.norm(
         locnorm = norm_impl(Q.realdata, Val(p), dims)
     end
 
-    mpiop = isfinite(p) ? MPI.SUM : MPI.MAX
+    mpiop = isfinite(p) ? (+) : max
```

Comment:
This might be problematic on platforms that don't support generic ops. I think Cray MPI and PowerPC

Reply:
On those platforms will this work for the standard types? Or do we have to use MPI.SUM and MPI.MAX here? This is really more for testing purposes, than real performance runs.

Reply:
No, this is fine. It will use MPI.SUM and MPI.MAX for MPI-defined types.